CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ the code was deployed.
 - Database columns `id` and `device_type` to table `devices` (CU-86791yyvg).
 - Database tables `buttons_vitals`, `buttons_vitals_cache`, `gateways`, `gateways_vitals`, and `gateways_vitals_cache`, replicating the tables used by Brave Buttons (CU-86791yyvg).
 - Country, country subdivision, and building type fields to create and edit client dashboard pages (CU-86791yyvg).
+- `consecutiveDoorOpenHeartbeatCount` variable which tracks the number of heartbeats in a row a door has been left open (CU-86794et47).
```

Comment:
Please move these lines into the [Unreleased] header of the CHANGELOG.md. The other changes are from the released version.

It should be like this:

```
# [Unreleased]

## Added

- (your changes)

# [10.9.0] - XXXX-XX-XX

...
```

Reply:
fixed in latest push